### DISCUSSION TITLE
Bots don't crouch when attacking with knife

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#13776] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Also unstable 1.4.3.0 doesn't have watermark

https://github.com/FakeFishGames/Barotrauma/assets/122838333/4916945c-8edb-46b7-a3c0-185beb882216

### Reproduction steps

control one of the guards
giveaffliction "Husk infection" 100, unpause, kill
See how other guards deal with this situation

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.3.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by AzureNocte** [comment]
I noticed this too since 1.4.0.0, also they can't crouch to weld hull breaches when needed to.

**Comment by itchyOwl** [comment]
Can repro, looking into it.

**Comment by itchyOwl** [comment]
Converted into an issue ticket: #13780

**Comment by Rokvach** [comment]
Issue has been fixed and tested, closing discussion.
